test/base/lane_test.cc

```diff
@@ -35,17 +35,29 @@
 #include <maliput/common/assertion_error.h>
 #include <maliput/math/vector.h>
 #include <maliput/test_utilities/maliput_math_compare.h>
+#include <maliput/test_utilities/maliput_types_compare.h>
 
 #include "maliput_sparse/geometry/line_string.h"
 
+#define IsLanePositionResultClose(lpr_a, lpr_b, tolerance)                                           \
+  do {                                                                                               \
+    EXPECT_TRUE(IsLanePositionClose(lpr_a.lane_position, lpr_b.lane_position, tolerance));           \
+    EXPECT_TRUE(IsInertialPositionClose(lpr_a.nearest_position, lpr_b.nearest_position, tolerance)); \
+    EXPECT_NEAR(lpr_a.distance, lpr_b.distance, tolerance);                                          \
+  } while (0);
+
```

Comment:
Can we move it to maliput_types_compare.h?

Reply:
Done https://github.com/maliput/maliput/pull/522